active/personpicture/PersonPictureAPI.md

```diff
@@ -0,0 +1,40 @@
+# Background
+
+The best practice for controls is that the XAML file includes all UI related colors, sizes and visual states.
+XAML markup has expressiveness limitations that require the controls to do some calculations or value conversions
+in code-behind and expose those results to the XAML to bind from. The pattern we have established for that is 
+having a TemplateSettings property on the control of type <I>Foo</I>Control, where <I>Foo</I> is the name of 
+the control. See [this doc page](https://docs.microsoft.com/en-us/windows/uwp/xaml-platform/template-settings-classes)
+for more details.
+
+# Description
+
+PersonPicture.TemplateSettings and the new type PersonPictureTemplateSettings are needed to help move some

```

Comment:
We're trying to use specs as a draft for what will go on docs.microsoft.com.

I think this is background info that could go in the Background section (which doesn't go to DMC), and the description (which goes to DMC) could be something like:

> Provides calculated values that can be referenced as TemplatedParent sources when defining templates for a PersonPicture control. Not intended for general use.

Reply:
Sounds good, took that suggestion.